### PR TITLE
Tigers/user journey

### PR DESCRIPTION
**Pull request by ramoshka**
Add call to start user journey so marketing team could send emails from ExactTarget.
API docs are [here](https://developer.salesforce.com/docs/atlas.en-us.noversion.mc-apis.meta/mc-apis/postEvent.htm).
Also tests are fixed not to break on `Faraday` update.
Solves Shopify/handshake-prediscovery-tigers#135

### CODE REVIEW

**Review by andreadipersio — APPROVED** [review]
LGTM, I merged api-v2 btw so you can merge this into master.

**Review by dariusshop — APPROVED** [review]
👏🏻 👏🏻 👏🏻

File: lib/exact_target_rest/support/interaction_event_params.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module ExactTargetRest
+  # Transformer for InteractionEvent payload
+  class InteractionEventParams
+    def initialize(*top_level_fields, snake_to_camel: true)
```
Comment (donatas-ai) [inline lib/exact_target_rest/support/interaction_event_params.rb]: Sorry for my amateurishness with Ruby :) does this mean that `snake_to_camel` is not used?

Comment (ramoshka) [reply lib/exact_target_rest/support/interaction_event_params.rb]: You're totally correct, will create PR for this

**Pull request merged**